desktop-src/dataxchg/html-clipboard-format.md

```diff
@@ -21,244 +21,346 @@ ms.date: 05/31/2018
+The selection, as delimited by `StartSelection` and `EndSelection`, is shown in bold.
 
-            <!-- EndFragment-->
+### Scenario 3 - Pasting a fragment of an ordered list `<ol>` into plain text
 
-            </OL>&lt;/BODY&gt;&lt;/HTML&gt;
+<!-- TODO: If a user copies a middle subsection of an ordered list, and then pastes it elsewhere, should the values of those bullet-points be preserved? If so, then how should the DOM represent it? -->
 
-            The selection, as delimited by StartSelection and EndSelection, is show in bold.
+Assume the following HTML text:
 
-        -   If this fragment is now pasted into an empty document, the following HTML will be created:
+> ```
+> <BODY><OL TYPE="a"><LI>Item 1<LI>Item 2<LI>Item 3<LI>Item 4<LI>Item 5<LI>Item 6</OL></BODY>
+> ```
 
-            &lt;BODY&gt;<OL TYPE = a><LI>Item 3<LI>Item 4<LI>Item 5</OL>&lt;/BODY&gt;
+This will appear as follows:
 
-        -   Appearing as:
-            1.  Item 3
-            2.  Item 4
-            3.  Item 5
+<!-- or just let Markdown do its own thang?
+> <ol>
+> 	<li>Item 1</li>
+> 	<li>Item 2</li>
+> 	<li>Item 3</li>
+> 	<li>Item 4</li>
+> 	<li>Item 5</li>
+> 	<li>Item 6</li>
+> </ol>
+ -->
 
-4.  Pasting a partially selected region.
-    -   -   HTML text:
+1.  Item 1
+2.  Item 2
+3.  Item 3
+4.  Item 4
+5.  Item 5
+6.  Item 6
 
-            <P> IE4/MSHTML is a WYSIWYG Editor that supports :<UL><LI>Cut<LI>Copy<LI>Paste</UL><P>This is a Great Tool !
 
-        -   Appears as:IE4/MSHTML is a WYSIWYG Editor that supports :
-            -   -   Cut
-                -   Copy
-                -   Paste
+<!-- (TODO: screenshot? but is that really necessary for markup this simple?) -->
 
-        -   The user selects from "WYSIWYG" until "Cop".The following HTML is in the clipboard:
 
-            <DOCTYPE...>&lt;HTML&gt;&lt;BODY&gt;
+#### How MSHTML handles copying a substring of HTML
 
-            <!-- StartFragment-->
+1. The user makes a text selection from the start of **item 3**, through **item 4**, and to the end of **item 5**. The user invokes the Copy command.
+2. The following HTML is in the clipboard (line breaks added for clarity) - the precise location of the `<!--Star/EndFragment -->` comments depends on how the user handled their browser's text selection logic:
 
-            <P>
 
-            **WYSIWYG Editor, which supports**
+```
+<html>
+<body>
+<ol>
+<!-- StartFragment-->
+<li>Item 3</li>
+<li>Item 4</li>
+<li>Item 5</li>
+<!-- EndFragment-->
+</ol>
+</body>
+</html>
+```
 
-            **<UL><LI>Cut<LI>Cop**
+If this fragment were to now be pasted into an empty document, the following HTML will be created:
 
-            </UL>
+```
+<body>
+<ol>
+<li>Item 3</li>
+<li>Item 4</li>
+<li>Item 5</li>
+</ol>
+</body>
+```
 
-            <!-- EndFragment-->
+This will appear as follows:
 
-            &lt;/BODY&gt;&lt;/HTML&gt;The selection, as delimited by StartSelection and EndSelection, is shown in bold.
+> 1.  Item 3
```

Comment:
I suggest to decide for markdown or not and use it consistently

Reply:
Agreed. Thanks again for the review and suggestions.